plugins/aea-test-autonomy/aea_test_autonomy/docker/registries.py

```diff
@@ -40,7 +40,13 @@
 GNOSIS_SAFE_MULTISIG = "0xA51c1fc2f0D1a1b8494Ed1FE312d7C3a78Ed91C0"
 SERVICE_REGISTRY = "0x998abeb3E57409262aE5b751f60747921B33613E"
 SERVICE_MANAGER = "0x70e0bA845a1A0F2DA3359C97E0285013525FFC49"
-SERVICE_MULTISIG = "0x0E801D84Fa97b50751Dbf25036d067dCf18858bF"
+GNOSIS_SAFE_MASTER_COPY = "0x4826533B4897376654Bb4d4AD88B7faFD0C98528"
+GNOSIS_SAFE_PROXY_FACTORY = "0x99bbA657f2BbC93c02D617f8bA121cB8Fc104Acf"
+GNOSIS_SAFE_MULTISIG = "0x0E801D84Fa97b50751Dbf25036d067dCf18858bF"
+GNOSIS_SAFE_MULTISIG_WITH_SAME_ADDRESS = "0x8f86403A4DE0BB5791fa46B8e795C547942fE4Cf"
+GNOSIS_SAFE_MULTISEND = "0x9d4454B023096f34B160D6B654540c56A1F81688"
+SERVICE_MULTISIG_1 = "0xD8dE647170163a981bb3Fdb2063583eAcF7D55AC"
+SERVICE_MULTISIG_2 = "0x070fC9F640b0cC01ab0159d23f09f8936077C9e9"
```

Comment:
These can be used in various tests and dev setups, Right now there are no places where we use them since they are recently introduced. I'll check if we use these values anywhere and replace them with these constants